### PR TITLE
docs: add example for: awscc_ec2_prefix_list

### PR DESCRIPTION
**Pull request by catlike**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_ec2_prefix_list/ec2_prefix_list_ipv4.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  max_entries      = 5
+  prefix_list_name = "example-ipv4-prefix-list"
+  entries = [{
+    cidr        = "10.10.0.0/16"
+    description = "example network"
+    },
+    { cidr = "192.168.2.8/32"
+    description = "example host" }
+  ]
+}
```
Comment (wellsiau-aws) [inline examples/resources/awscc_ec2_prefix_list/ec2_prefix_list_ipv4.tf:12]:
```suggestion
resource "awscc_ec2_prefix_list" "example_prefix_list" {
  address_family   = "IPv4"
  max_entries      = 5
  prefix_list_name = "example-ipv4-prefix-list"
  entries = [
    {
      cidr        = "10.10.0.0/16"
      description = "example network"
    },
    {
      cidr        = "192.168.2.8/32"
      description = "example host"
    }
  ]
}
```

File: examples/resources/awscc_ec2_prefix_list/ec2_prefix_list_ipv6.tf [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  max_entries      = 5
+  prefix_list_name = "example-ipv6-prefix-list"
+  entries = [{
+    cidr        = "2001:db8::/32"
+    description = "example network"
+    },
+    { cidr = "2001:db8:abcd:0012::0/128"
+    description = "example host" }
+  ]
+}
```
Comment (wellsiau-aws) [inline examples/resources/awscc_ec2_prefix_list/ec2_prefix_list_ipv6.tf:12]:
```suggestion
resource "awscc_ec2_prefix_list" "example_ipv6_prefix_list" {
  address_family   = "IPv6"
  max_entries      = 5
  prefix_list_name = "example-ipv6-prefix-list"
  entries = [
    {
      cidr        = "2001:db8::/32"
      description = "example network"
    },
    {
      cidr        = "2001:db8:abcd:0012::0/128"
      description = "example host"
    }
  ]
}
```

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
Small minor suggestion to tidy-up the formatting, otherwise looks good. Thanks!

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thanks!

**Review by wellsiau-aws — APPROVED** [review]

**Comment by marcosentino** [comment]
@catlike Thanks for the contribution!

**Pull request merged**